eos-migrate-image-defaults

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -ex
+
+old_path=/var/eos-image-defaults
+new_path=/var/lib/eos-image-defaults
+dconf_profile=/etc/dconf/profile/user
+
+if [ -d ${old_path} -a ! -h ${old_path} -a ! -e ${new_path} ]; then
```

Comment:
Ah - because we're doing 3 operations which aren't atomic, it's intentionally written to not early return so that it's idempotent, so that if it's interrupted at any stage, running it again will complete the migration - so if it gets messed up somehow (admittedly unlikely), you can just run the script again. An early return would prevent the later parts from running.